prometheus_juju_backup_all_exporter/utils.py

```diff
@@ -20,81 +20,67 @@ class BackupStats:
     """A class representing backup statistic file."""
 
     def __init__(self, config):
-        """Initialize class."""
-        self.set_properties(Path(config.backup_path, "backup_stats.json"))
-
-    @property
-    def duration(self):
-        return self._duration
-
-    @property
-    def status_ok(self):
-        return self._status_ok
-
-    @property
-    def result_code(self):
-        return self._result_code
-
-    def set_properties(self, stats_file):
-        """Set the instance properties."""
+        """Initialize and set instance properties."""
+        self._duration = 0
+        self._status_ok = 0
+        self._result_code = 3  # unknown
```

Comment:
not-blocking: Maybe we can define these values as constants?

Reply:
Done